packages/next/build/webpack-config.ts

```diff
@@ -1817,8 +1817,8 @@ export default async function getBaseWebpackConfig(
           {
             appDir: dir,
             esmExternals: config.experimental.esmExternals,
-            staticImageImports: !config.images.disableStaticImages,
```

Comment:
Why is this one removed? We still allow this config option

https://nextjs.org/docs/api-reference/next/image#disable-static-imports

Reply:
It wasn't used in the plugin and just didn't get cleaned up.

Reply:
For more context this was removed in favor of the `reasons` handling here https://github.com/vercel/next.js/blob/3a6cd49c317c6b64e379161ba7d7fd27c4db2b64/packages/next/build/webpack/plugins/next-trace-entrypoints-plugin.ts#L367-L369